src/commands/uplink/GetShortNameResponse.ts

```diff
@@ -0,0 +1,111 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import CommandBinaryBuffer, {IObis} from '../../CommandBinaryBuffer.js';
+import {UPLINK} from '../../constants/directions.js';
+
+
+/**
+ * IGetShortNameResponseParameters command parameters
+ */
+interface IGetShortNameResponseParameters {
+    obis: IObis,
+    shortNameList: Array<number>
+}
+
+const COMMAND_ID = 0x01;
+
+const examples: TCommandExampleList = [
+    {
+        name: 'two short names for OBIS code 8.0.9.1 - local time',
```

Comment:
do we need to mention time here?

Reply:
yes, of course

a lot of OBIS code exists, why we can't help and remind about it

Reply:
need clarification

Reply:
fixed, OBIS interface reworked, "group" word deleted